.release-please-manifest.json

```
@@ -91,5 +91,6 @@
   "plugins/source/stripe+FILLER": "0.0.0",
   "plugins/source/mixpanel": "1.0.0",
   "plugins/source/hubspot": "1.0.0",
```

```suggestion
  "plugins/source/mixpanel": "1.0.0",
  "plugins/source/mixpanel+FILLER": "0.0.0",
  "plugins/source/hubspot": "1.0.0",
  "plugins/source/hubspot+FILLER": "0.0.0",
```

Let's add some for Mixpanel and Hubspot while we're here